### PR TITLE
Avoid posting review if comments are empty

### PR DESCRIPTION
**Pull request by ikerexxe**
Test results can be reviewed in https://github.com/ikerexxe/test-1/pull/2

### CODE REVIEW

File: cmd/action-cppcheck/main.go [hunk 1]
```diff
@@ -129,12 +129,16 @@ func main() {
...
 		}
 	}
-	_, _, err = client.PullRequests.CreateReview(context.Background(), owner, repo, pullID,
-		&github.PullRequestReviewRequest{
-			Event:    github.String(string(event)),
-			Body:     &body,
-			Comments: comments,
-		})
+
+	if len(comments) > 0 {
```
Comment (myml) [inline cmd/action-cppcheck/main.go]: Need comment_result is false
like
```
if comments == 0 && !comment_result{
return
}
```

**Review by myml — CHANGES_REQUESTED** [review]

File: cmd/action-cppcheck/main.go (hunk 1 above)
Comment (ikerexxe) [reply cmd/action-cppcheck/main.go]: There's no need for that. It's already implied that if `comments` is empty, then `comment_result` is false.

Comment (myml) [reply cmd/action-cppcheck/main.go]: Comment result is optional. If comments is empty and comment result is true, should post good words.

Comment (ikerexxe) [reply cmd/action-cppcheck/main.go]: You are right. I've updated the code accordingly.

**Comment by ikerexxe** [comment]
I've updated the code and rerun the check in https://github.com/ikerexxe/test-1/pull/2

File: cmd/action-cppcheck/main.go [hunk 2]
```diff
@@ -129,12 +129,20 @@ func main() {
...
-			Event:    github.String(string(event)),
-			Body:     &body,
-			Comments: comments,
-		})
+
+	if len(comments) == 0 && !commentResult{
+		return
+	}
+
+	if len(comments) > 0 {
```
Comment (myml) [inline cmd/action-cppcheck/main.go]: This 'if' needs to be removed to send "good comment"

**Review by myml — CHANGES_REQUESTED** [review]

File: cmd/action-cppcheck/main.go (hunk 2 above)
Comment (ikerexxe) [reply cmd/action-cppcheck/main.go]: Done

**Comment by myml** [comment]
thank for you contribution!

**Pull request merged**